nomt/src/beatree/ops/update/branch_updater.rs

```diff
@@ -468,7 +468,7 @@ impl BranchBulkSplitter {
 }
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
```

Comment:
I'll recommend a small refactor here: extract the relevant code to a `test_helpers` module.

It's a little rough to expose `mod tests` publicly. Not a huge deal, but not great style.